src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableCheckSwitchExpressions.java

```diff
@@ -7,6 +7,13 @@
  * tokens = {VARIABLE_DEF}
  */
 public class InputFinalLocalVariableCheckSwitchExpressions {
+    private static final int staticValue = 2;
```

Comment:
Please extend Input for default config

Reply:
Sorry, I don't understand what you mean. Could you please explain?

Reply:
if you pay attention to comment above class you will see that there is definition of Check config.

your bug is for default config of Check, without any modification in properties.
please create https://github.com/checkstyle/checkstyle/tree/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable
new file InputFinalLocalVariableCheckDefaultSwitchExpressions.java and use it for default config configuration., so new UT method should be created to use new file.

Reply:
@romani this is the default config; during summer I thought that it might be a good idea to list the config regardless of whether it is default or not, which is why this file doesn't say `default`.  At some point in the future, the default could change, and then we would have to either update configs at the top of the input files or ... 

 I do agree that a new input and UT should be created.

Reply:
> new file InputFinalLocalVariableCheckDefaultSwitchExpressions.java

Since the existing input has no switch *expressions* on RHS of variable definition, we could just call it  `InputFinalLocalVariableCheckSwitchAssignment.java` or something.

Reply:
default config should be `/* Config: default` 

definition of default is useful during review agree, I really hope one day we will convert UTs to be aligned with BDD concepts. It is project for future GSoCs.

Reply:
fixed. I have added a separate UT, as well as `InputFinalLocalVariableCheckSwitchAssignment.java`. Thank you for your guidance.